subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -248,6 +248,15 @@ config LWM2M_CLIENT_UTILS_RAI
 	  When AS RAI is configured, device may indicate that no further data is
 	  expected in the near future and the connection may be released.
 
+config LWM2M_CLIENT_UTILS_CONNEVAL
```

Comment:
I wonder if this option/API should include `LTE` keyword, like `LTE_CONNEVAL`. With other networking technologies on the table now, current name might be too generic.